portality/forms/application_forms.py

```diff
@@ -1792,6 +1792,26 @@ class FieldDefinitions:
         ],
         "help" : {
             "render_error_box": False
+        },
+        "contexts": {
+            "admin": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                    "full_contents"  # ~~^->FullContents:FormWidget~~
```

Comment:
`tagentry` style is gone, is it user required ?

Reply:
Yes, tagentry is required, @amdomanska if you override the widgets, you have to override them all, it won't merge the lists